snapcraft/tests/unit/commands/test_export_login.py

```diff
@@ -132,3 +129,50 @@ def test_failed_login_with_invalid_credentials(self, mock_login):
         self.assertThat(result.output, Contains(
             storeapi.constants.INVALID_CREDENTIALS))
         self.assertThat(result.output, Contains('Login failed.'))
+    store = storeapi.StoreClient()
+    snap_name = store.get_snap_name_for_id('edge123')
```

Comment:
This is weird, the indentation is not right. Can you explain what are you trying to do here? We might be able to guide you better if you document your intention and reasons on the pull request description.